.travis.yml

```diff
@@ -113,10 +113,10 @@ script:
   # triggers the nightly build.
   - |
     if [[ "${METABENCH_NIGHTLY_BUILD}" == "true" && "${CMAKE_GENERATOR}" == "Unix Makefiles" ]]; then
-      cmake --build build --target benchmarks
+      cmake --build build --target benchmarks &&
```

Comment:
Shouldn't the script already exit as soon as a command returns with a non zero status? This is strange, do you know why we're removing everything when it fails?